vendor-extra/LiberoPatternsBundle/src/Resources/views/content-header.html.twig

```diff
@@ -4,6 +4,16 @@
 
   {%- block body -%}
 
+    {%- with categories|default({}) only -%}
```

Comment:
Not blocking: Have we considered automatically introducing the latest patterns in line with them being deployed in pattern library to reduce the delta for these PR's?

Reply:
We will do, not sure when's the best time. The issue at the minute is that the patterns were merged in a different order to how they're being implemented. Given that things are unblocked hopefully we can keep the order reasonably consistent which will mitigate it.